kapitan/utils.py

```diff
@@ -471,3 +475,41 @@ def make_request(source):
     else:
         r.raise_for_status()
     return None, None
+
+
+def unpack_downloaded_file(file_path, output_path, content_type):
```

Comment:
I don't understand the need for this. Can't we use/extend https://github.com/deepmind/kapitan/blob/7b3a4af419e9716faa11d76a6f0bc258e1b33b8b/kapitan/dependency_manager/base.py#L132 instead?

Reply:
I thought of shifting the code from `kapitan/kapitan/dependency_manager/base.py` to the utils section as unpacking a downloaded file might be useful while implementing other functionalities in the future?